### DISCUSSION TITLE
Dynamic nested forms creation on SunbirdEd-forms.

### DISCUSSION THREAD

**Discussion by bharathravi-in** [Sunbird-Ed/Community#470] [General Discussion]
hi team i need a walkthrough the existing sb-forms creation and need some suggestions for the new implementation of nested forms dynamically.

@rajeevsathish @vinukumar-vs

**Comment by Krishnaj20** [comment]
@BharathTarento - can you let us know the use cases where these dynamic forms will be used?

**Reply by bharathravi-in** [reply]
@Krishnaj20  we want to add a criteria (RFC - Dynamic QuestionSet) for the section to get dynamic questions on load of question set in a player.

**Reply by Krishnaj20** [reply]
@rajeevsathish @chitranshu-keshav - Please plan this demo as part of the next Sunbird community fortnightly meeting.

**Reply by Krishnaj20** [reply]
@chitranshu-keshav - Can you plan this to be covered by @rajeevsathish in the next Sunbird Community fortnightly meeting?

**Reply by Krishnaj20** [reply]
@chitranshu-keshav - is this taken care of?

**Comment by chitranshu-keshav** [comment]
@Krishnaj20 this will be planned post 2nd June post completion of 6.0 development

**Reply by chitranshu-keshav** [reply]
@rajeevsathish since there is a delay in 6.0.0 when can this be planned next?

**Comment by ashokreddy1208** [comment]
@bharathravi-in let us know if you still need walkthrough session on sb-forms? can you please confirm and update this thread
